docs/Release_Notes.gfs.v16.1.8.txt

```diff
@@ -0,0 +1,128 @@
+GFS V16.1.8 RELEASE NOTES
```

Comment:
Why adding the old release notes in this PR?

Reply:
Will look at cleaning out old release notes from v16 branches later.